conanfile.txt

```diff
@@ -1,22 +1,19 @@
 [requires]
-boost/1.71.0
+boost/1.83.0
 
 [generators]
 cmake_find_package
 json
 
 [options]
 boost:header_only=True
-boost:without_atomic=True
```

Comment:
it wouldn't install boost without having these libraries as well. it's not compiling anything though, I guess it's just optional dependencies it now expects for some reason